### ISSUE TITLE
Add option to hide 'Recently Added' 

### ISSUE THREAD

**Issue by sohnemann** [janeczku/calibre-web#1576]
**Is your feature request related to a problem? Please describe.**
I'm unable to hide the section _Recently Added_ from the Guest or any other user.
I would like to only show public shelves in the guest view, but not my recently added books
which should remain private.

**Describe the solution you'd like**
Add a checkbox under _Admin_ -> _Users_ -> _Guest / user_ -> "Show recent books". 
When unticked, only show public shelves but nothing else.

**Describe alternatives you've considered**
Unfortunately I didn't find any real alternative to this feature, 
as adding tags to all books is error prone and a lot of work.

**Additional context**
Here's a screenshot of how it looks right now without the option:
<img src="https://i.imgur.com/zg1npR8.png" alt="Screenshot of Guest user settings" />

**Comment by kenjibailly** [comment]
I'm wondering if there's a workaround already?

**Comment by OzzieIsaacs** [comment]
I answered in another issue (can't remember which one) that removing this menu item will cause a lot of trouble, as it's the home/root folder of calibre-web. So there is no workaround for this and there will be none in the near future. Meanwhile the sorting order is saved on a per user base, so it's no longer recently added, but books sorted according to title, publishing date, library adding date, and authors. I renamed the description to "books", not the smartest name, but I have no better idea yet.
